### ISSUE TITLE
FR: Copy percentage complete in `sup` command can be under 100% when finished

### ISSUE THREAD

**Issue by ggunson** [github/gh-ost#209]
An example `sup` from last week:

```
Copy: 666913837/669324238 99.6%; Applied: 5521513; Backlog: 11/100; Time: 26h56m48s(total), 26h26m27s(copy); streamer: mysql-bin.010108:717250323; ETA: postponing cut-over 
```

The row copy had completed (since it was `postponing cut-over`) but according to the "Copy" status it was at `99.6%`. The count discrepancy could make sense since the row copy didn't start until half an hour after the `SELECT count(*) from table` started; maybe this will be less of an issue due to concurrent counts in https://github.com/github/gh-ost/pull/196, but I expect it'll still happen in some cases.

My suggestion would be to make it say `100%` if the row copying is complete; otherwise a user might think that the copying isn't completed yet (despite "postponing cut-over"), or that rows were lost in the copying and it's not safe to do the table swap. Or any other suggestion for `sup` output that clarifies the status and assures the user that all is well.

**Comment by shlomi-noach** [comment]
:+1: good idea! 
The heuristic for the on-going updated row estimate can be improved, and is not 100.0% accurate, which explains the diff.
Will hard code `100%` once rowcopy is known to be complete.

**Comment by shlomi-noach** [comment]
addressed by #213
